src/drivers/SpiMaster.cpp

```diff
@@ -153,7 +153,7 @@ void SpiMaster::OnEndEvent() {
 void SpiMaster::OnStartedEvent() {
 }
 
-void SpiMaster::PrepareTx(const volatile uint32_t bufferAddress, const volatile size_t size) {
+void SpiMaster::PrepareTx(const uint32_t bufferAddress, const size_t size) {
```

Comment:
Why remove volatile? Isn't that needed anymore with C++20?

Reply:
Yes, exactly, that's another change I had to make when I switched to C++20. Here's the error from the compiler: 

```
/home/jf/git/PineTime/src/drivers/SpiMaster.cpp:156:51: error: 'volatile'-qualified parameter is deprecated [-Werror=volatile]
  156 | void SpiMaster::PrepareTx(const volatile uint32_t bufferAddress, const volatile size_t size) {
      |                           ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~
/home/jf/git/PineTime/src/drivers/SpiMaster.cpp:156:88: error: 'volatile'-qualified parameter is deprecated [-Werror=volatile]
  156 | void SpiMaster::PrepareTx(const volatile uint32_t bufferAddress, const volatile size_t size) {
      |                                                                  ~~~~~~~~~~~~~~~~~~~~~~^~~~
```